bb-new.html

```diff
@@ -0,0 +1,296 @@
+          <p>We’re known for secure handsets that changed the world, and how we work. Today, we’re reimagining protection, again, with artificial intelligence and machine learning. We make everything more secure—and you, more productive.</p>
+          <ul>
+            <li class="sec-list-items"><a href="#">CYBERSECURITY</a></li>
+            <li class="sec-list-items"><a href="#">CRITICAL EVENT MANAGEMENT</a></li>
+            <li class="sec-list-items"><a href="#">INTERNET OF THINGS (loT)</a></li>
+          </ul>
+        </div>
+        <div class="col">
+          <img src="/images/men-holding-laptop-on-baconey.jpg" alt="men holding laptop outside on a baconey">
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <img src="/images/men-back-sitting-on-chair.jpg" alt="men back sitting on a chair facing a laptop">
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <img src="/images/security-icon.jpg" alt="blackberry security icon">
+          <h2>WE’VE NEVER BEEN MORE CONNECTED.OR MORE VULNERABLE.</h2>
+          <p>Cybersecurity has failed to keep up, because it fails to look ahead. Our intelligent security pairs artificial intelligence with machine learning to proactively protect your system from cyberthreats. It’s time to protect, prevent and respond.</p>
+          <button syle="display: inline-block">CYBER SUITE</button>
+          <button style="display: inline block">SPARK SUITE</button>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <img src="/images/security-icon-2.jpg" alt="blackberry security icon">
+          <h2>ALWAYS-ON. ALWAYS PROTECTED</h2>
+          <p>In times of crisis, mobility and secure communications are essential. Our critical event management solutions are built on our trusted reputation of absolute reliability. We don’t just keep communications safe—we keep people safe.</p>
+          <button syle="display: inline-block">BLACKBERRY ATHOC</button>
+          <button style="display: inline block">BLACKBERRY ALERT</button>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <img src="/images/astronaut.jpg" alt="astronaut floting">
+        </div>
+      <div class="row">
+        <div class="col">
+          <img src="/images/security-icon-3.jpg" alt="blackberry security icon">
+          <h2>FOR WHERE YOU ARE NOW. AND WHERE YOU GO NEXT.</h2>
+          <p>Mission-critical systems can’t afford to fail. We’re protecting the Internet of Things which includes over 500 million endpoints—like robotic neurosurgical devices and over 175 million cars on the road today. As the number of users, devices and endpoints grow, so does the need for intelligent security.</p>
+          <button syle="display: inline-block">BLACKBERRY QNX</button>
+        </div>
+      </div>         
+      </div>
```

Comment:
You can tell there are indentation issues from the very fact that these closing div tags are shown here on the same level.